Source/Scene/Cesium3DTileset.js

```diff
@@ -1285,11 +1336,21 @@ define([
                 // With replacement refinement, if the tile's SSE
                 // is not sufficient, its children (or ancestors) are
                 // rendered instead
+                var useChildrenBoundUnion = t._optimizations.checkChildrenWithinParent(t, true);
 
+                var childrenVisibility;
+                
                 if ((sse <= maximumScreenSpaceError) || (childrenLength === 0)) {
                     // This tile meets the SSE so add its commands.
                     // Select tile if it's a leaf (childrenLength === 0)
-                    selectTile(tileset, t, fullyVisible, frameState);
+                    if (useChildrenBoundUnion) {
```

Comment:
Are you sure this is the right place to add this code?  I was expecting a place that use to have
```
checkThisTilesVolume
```
to be replaced with
```
checkTheUnionOfTheTilesChildsVolumes
```